shuup/admin/modules/products/views/list.py

```diff
@@ -103,9 +105,34 @@ class ProductListView(PicotableListView):
     toolbar_buttons_provider_key = "product_list_toolbar_provider"
     mass_actions_provider_key = "product_list_mass_actions_provider"
 
+    def __init__(self):
+        def get_suppliers_column(iterable):
+            return first([col for col in iterable if col.id in ["suppliers", "shopproduct_suppliers"]], default=None)
+
+        def get_suppliers_filter():
+            return TextFilter(filter_field="suppliers__name", placeholder=_("Filter by supplier name..."))
+
+        if settings.SHUUP_ENABLE_MULTIPLE_SUPPLIERS and not get_suppliers_column(self.default_columns):
+            self.default_columns.append(
+                Column(
+                    "suppliers",
+                    _("Suppliers"),
+                    display="format_suppliers",
+                    ordering=8,
+                    filter_config=get_suppliers_filter(),
+                )
+            )
+        super(ProductListView, self).__init__()
```

Comment:
are we officially dropping py 27?

Reply:
Not just yet.

Reply:
so this needs fixing

Reply:
how is this not py27 compatible?

Reply:
:man_facepalming: that is right, Monday...